app/Http/Resources/RoomCollection.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\ResourceCollection;
+
+class RoomCollection extends ResourceCollection
+{
+
+  public static $wrap = null;
+
+
+  /**
+     * Transform the resource collection into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return parent::toArray($request);
+    }
```

Comment:
Hey maybe we can remove function? Seems sonarcube picked it up as a code smell.

Reply:
Yeah worked like a charm.